### PR TITLE
Update dg find

### PR DESCRIPTION
**Pull request by TheRealRavager**
My DG and PPP

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -623,55 +623,76 @@ This section will explain the reason for having an aspect.
...
-TODO
+The find feature is facilitated by `PersonContainsPrefixesPredicate` and `FindCommand`. +
 
 ===== PersonContainsPrefixesPredicate
+`PersonContainsPrefixesPredicate` implements `Predicate` and is used to test whether a person's attributes match exactly
+with at least one of the input keywords in each attribute category.
 
-TODO
+===== FindCommand
+`FindCommand` extends extends `Command` with a `predicate` that is used to determine whether a given `person` should be
```
Comment (C-likethis123) [inline docs/DeveloperGuide.adoc]: typo, double extends

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -623,55 +623,76 @@ This section will explain the reason for having an aspect.
...
-Step 2: `argMultimap` is used to initialize a `PersonContainsPrefixesPredicate` object: `predicate`.
+Step 2: A list of exact matches is obtained by filtering [blue]`allPersons` using
+[blue]`PersonContainsPrefixesPredicate`.
 
-Step 3: `predicate` is used to initialize a `FindCommand` object. `#execute` is then called on it.
+Step 3: A list containing all persons except exact matches is obtained and fuzzy matching is performed on it.
 
-Step 4: `#execute` calls the `#updateFilteredPersonList` of the current `Model` which causes it's
-`FilteredList<Person>`: `filteredPersons` to automatically remove `persons` that fail `predicate#test`.
+Step 4: A list containing all persons except exact matches is obtained and fuzzy matching is performed on it.
```
Comment (C-likethis123) [inline docs/DeveloperGuide.adoc]: duplicate steps

File: docs/AboutUs.adoc [hunk 3]
```diff
@@ -30,7 +30,7 @@ Responsibilities: Ensure project deliverables are done in right format and submi
 
 === Iskandar Zulkarnaien Bin Suhaini
 image::therealravager.png[width="150", align="left"]
-{empty}[https://github.com/TheRealRavager[github]] [<<therealravager #, portfolio>>]
+{empty}[https://github.com/TheRealRavager[github]] [<<TheRealRavager#, portfolio>>]
```
Comment (SimNee) [inline docs/AboutUs.adoc]: if i am not wrong, I think the name of your ppp needs to be in all small caps

File: docs/team/TheRealRavager.adoc [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+TIP: This section contains a helpful tip!
+
+`find` A grey highlight (called a mark-up) indicates that this is a command that can be typed into the command box
+and executed by the application.
+
+== Summary of contributions
+This section shows a summary of my coding, documentation and other helpful contributions to the team project.
+
+*Feature Implemeted*: I added the ability to search for persons based on their attributes.
```
Comment (C-likethis123) [inline docs/team/TheRealRavager.adoc]: Note the format, it should be 'major enhancement'

File: docs/team/TheRealRavager.adoc [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+than having to scroll through the entire list of persons in ORGA__Nice__.
+* Highlights: This feature performs fuzzy matching on the inputted name, NRIC and phone number to allow for relevant
+search results to be displayed even if the user makes a typo.
+
+*Code contributed*: Click https://nus-cs2103-ay1920s1.github.io/tp-dashboard/#search=TheRealRavager&sort=groupTitle&sortWithin=title&since=2019-09-06&timeframe=commit&mergegroup=false&groupSelect=groupByRepos&breakdown=false&tabOpen=true&tabType=authorship&tabAuthor=TheRealRavager&tabRepo=AY1920S1-CS2103T-T13-1%2Fmain%5Bmaster%5D[here]
+to view my code contributions.
+
+*Other contributions*:
+
+* NRIC Checksum Validation (Pull Request https://github.com/AY1920S1-CS2103T-T13-1/main/commit/36a54c909e8a621bacfb0626b0161179f6c3a2d6[#255])
```
Comment (C-likethis123) [inline docs/team/TheRealRavager.adoc]: You may wish to put this under the section 'Enhancements to existing features'

File: docs/team/TheRealRavager.adoc [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+* Dummy Data Generation (Pull Request https://github.com/AY1920S1-CS2103T-T13-1/main/commit/9707403685db8ad694d45602b892f3ddece18e5d[#172])
+* Git Workflow Management:
```
Comment (C-likethis123) [inline docs/team/TheRealRavager.adoc]: This could be under 'Project management'

File: docs/team/therealravager.adoc [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+students from the National University of Singapore. ORGA__Nice__ was developed as part of CS2103T, an introductory
+Software Engineering module. ORGA__Nice__ is a desktop application created to help hospital administration staff manage
+organ transplants. ORGA__Nice__ was developed with the assumption that they prefer typing compared to other forms of
+usage. As such, ORGA__Nice__ utilises a Command Line Interface - a text-based system where users type in commands to
+execute tasks rather than clicking buttons. As part of the module, ORGA__Nice__ is morphed from an existing project,
+https://se-education.org/addressbook-level3/[AddressBook-Level3], an address book management application.
+
+This is what ORGA__Nice__ looks like:
+
+.The graphical user interface for ORGANice
```
Comment (Yen-Peng) [inline docs/team/therealravager.adoc]: i think u missed italicizing this organice

**Pull request merged**